nano/node/node.cpp

```diff
@@ -1048,6 +1048,12 @@ stats (config.stat_config),
 vote_uniquer (block_uniquer),
 startup_time (std::chrono::steady_clock::now ())
 {
+	if (config.websocket_config.enabled)
+	{
+		websocket_server = std::make_shared<nano::websocket::listener> (*this, nano::tcp_endpoint (boost::asio::ip::address_v6::any (), config.websocket_config.port));
```

Comment:
The bind address should be configurable.

Reply:
Good idea, adding the same way as is done for RPC